tests/test_benchmark.py

```diff
@@ -210,3 +214,22 @@ def get(f):
             == get(ClassWithOracles.some_staticmethod)
             == N_a + N_b + N_c + N_class + N_child
         )
+
+
+def test_inst_evaluate_throws_type_error(rng):
+    with pytest.raises(
+        TypeError,
```

Comment:
please add the full type error string as well, like `TypeError("something went wrong with the type of something")`